node/src/chain_spec.rs

```diff
@@ -254,9 +247,8 @@ fn testnet_genesis(
 			changes_trie_config: Default::default(),
 		},
 		balances: canvas_runtime::BalancesConfig {
-			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
+			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(), /* TODO Remove in production */
```

Comment:
Not sure if this note is necessary. For a production chain (Kusama, Polkadot) we'd be using a different genesis function, and there we'd explicitly **not** endow any accounts

```suggestion
			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
```